webapp/sources/rudder/rudder-web/src/main/resources/logback.xml

```diff
@@ -416,6 +416,13 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
     <appender-ref ref="STDOUT" />
   </logger>
 
+  <!--
+    Log information about user authentication and user-related access to Rudder 
+    (rights, session, user management in general).
+  -->
+  <logger name="application.authentication" level="info" additivity="false">
+    <appender-ref ref="STDOUT" />
```

Comment:
you don't need the complicated part when you only give stdout, since it is the default appender. 
Plus for logger with parents (logback is like log4j & all hierachized by logger doted names), you inherite parent conf. 

So just `<logger name="application.authentication" level="info" />` is enough and likely the config behavior we want.

Reply:
Plus, we should comment it out by default, so that it follows the level of `application`, which is consistant with what we want

Reply:
Right, it is now commented by default to  "debug" level : https://github.com/Normation/rudder/pull/5883/commits/657e2d019d03ad1bb58d8ea53ba2a8fac022a398